content/community/videos.md

```diff
@@ -7,168 +7,32 @@ permalink: community/videos.html
-</tr>
-<tr>
-<th>03:08</th><td>The symbiotic relationship of ClojureScript and React</td>
-<th>29:20</th><td>React evolved through survival of the fittest at Facebook</td>
-</tr>
-<tr>
-<th>04:54</th><td>The history of React and why it was created</td>
-<th>30:15</th><td>Ideas for having state on server and client, using web sockets.</td>
-</tr>
-<tr>
-<th>09:43</th><td>Updating web page with React without using data binding</td>
-<th>32:05</th><td>React-multiuser - distributed shared mutable state using Firebase</td>
-</tr>
-<tr>
-<th>13:11</th><td>Using the virtual DOM to change the browser DOM</td>
-<th>33:03</th><td>Better debugging with React using the state transitions, replaying events</td>
-</tr>
-<tr>
-<th>13:57</th><td>Programming with React, render targets HTML, canvas, other</td>
-<th>34:08</th><td>Differences from Web Components</td>
-</tr>
-<tr>
-<th>16:45</th><td>Working with designers. Contrasted with Ember and AngularJS</td>
-<th>34:25</th><td>Notable companies using React</td>
-</tr>
-<tr>
-<th>21:45</th><td>JSX Compiler bridging HTML and React javascript</td>
-<th>35:16</th><td>Could a React backend plugin be created to target PDF?</td>
-</tr>
-<tr>
-<th>23:50</th><td>Autobuilding JSX and in browser tools for React</td>
-<th>36:30</th><td>Future of React, what's next?</td>
-</tr>
-<tr>
-<th>24:50</th><td>Tips and tricks to working with React, getting started</td>
-<th>39:38</th><td>Contributing and getting help</td>
-</tr>
-</table>
-
-### JavaScript Jabber Podcast
-
-[Pete Hunt](http://www.petehunt.net/) and [Jordan Walke](https://github.com/jordwalke) talked about React in [JavaScript Jabber 73](https://devchat.tv/js-jabber/073-jsj-react-with-pete-hunt-and-jordan-walke).
-<figure><a aria-label="link to play JavaScript Jabber Podcast episode 73" href="http://javascriptjabber.com/073-jsj-react-with-pete-hunt-and-jordan-walke/#content"><img src="../images/docs/javascript-jabber.png"></a></figure>
-
-<table width="100%">
-  <tr>
-    <th>01:34</th><td>Pete Hunt Introduction</td>
-    <th>23:06</th><td>Supporting Node.js</td>
-  </tr>
-  <tr>
-    <th>02:45</th><td>Jordan Walke Introduction</td>
-    <th>24:03</th><td>rendr</td>
-  </tr>
-  <tr>
-    <th>04:15</th><td>React</td>
-    <th>26:02</th><td>JSX</td>
-  </tr>
-  <tr>
-    <th>06:38</th><td>60 Frames Per Second</td>
-    <th>30:31</th><td>requestAnimationFrame</td>
-  </tr>
-  <tr>
-    <th>09:34</th><td>Data Binding</td>
-    <th>34:15</th><td>React and Applications</td>
-  </tr>
-  <tr>
-    <th>12:31</th><td>Performance</td>
-    <th>38:12</th><td>React Users Khan Academy</td>
-  </tr>
-  <tr>
-    <th>17:39</th><td>Diffing Algorithm</td>
-    <th>39:53</th><td>Making it work</td>
-  </tr>
-  <tr>
-    <th>19:36</th><td>DOM Manipulation</td>
-    <th></th><td></td>
-  </tr>
-</table>
+ - [Introduction to React](https://www.youtube.com/watch?v=XxVg_s8xAms&feature=youtu.be) (2013 - 1h20m) - Tom Occhino and Jordan Walke introduce React at Facebook Seattle.
```

Comment:
I think the embedded videos were kind of nice. I'm curious why you removed them?

Reply:
My only reasoning was to keep the cohesiveness between pages / sections. Most other sections had a bullet list layout for the links, so I thought I would keep it the same. I'm not married to the idea and certainly don't mind reverting to the embedded videos. Your call @bvaughn !

Reply:
I don't feel strongly, but I think the embedded videos might be a nice touch.